packages/qwik/src/cli/utils/templates.ts

```diff
@@ -58,7 +58,10 @@ export async function readTemplates(rootDir: string) {
 }
 
 function parseTemplatePath(path: string, type: string) {
-  const parts = path.split(`/${type}/`);
+  let parts = path.split(`/${type}/`);
+  if (parts.length == 1) {
```

Comment:
I don't think this is the best way to fix this. Can we just use file system separator?

```typescript
const path = require('path');
path.splipt(path.sep + type + path.sep);
```

Reply:
Of course